util/test/tests/GL/GL_Backdoor_Context.py

```diff
@@ -0,0 +1,28 @@
+import renderdoc as rd
+import rdtest
+
+
+class GL_Backdoor_Context(rdtest.TestCase):
+    demos_test_name = 'GL_Backdoor_Context'
+
+    def check_capture(self):
+        
+        draw = self.find_draw("Draw")
+
+        self.controller.SetFrameEvent(draw.eventId, False)
+
+        pipe: rd.PipeState = self.controller.GetPipelineState()
+
+        texs: List[rd.BoundResourceArray] = pipe.GetReadOnlyResources(rd.ShaderStage.Fragment)
+        
+        id = texs[0].resources[0].resourceId
+
+        tex_details = self.get_texture(id)
+
+        for y in range(0, tex_details.height):
+            for x in range(0, tex_details.width):
+                self.check_pixel_value(id, x, y, [1.0, 0.5, 0.25, 1.0], 0.005)
```

Comment:
Checking every pixel like this is inefficient. I'd suggest either taking a few fixed samples (say each corner, and the centre), or if you think it's important enough to check every pixel else do an image compare against a png. Since it's a single colour the png will be tiny.